{"name": "test_divisible_by_6", "status": "failed", "statusDetails": {"message": "AssertionError: assert 3 == 0\n  -3\n  +0", "trace": "input_value = 39\n\n    def test_divisible_by_6(input_value):\n>       assert input_value % 6 == 0\nE       assert 3 == 0\nE         -3\nE         +0\n\ntest_div_by_3_6.py:13: AssertionError"}, "start": 1573041325453, "stop": 1573041325454, "uuid": "b8ae62bc-99a4-418b-83b9-6125f0ba8775", "historyId": "32b8478b746a25bb1aabbecba9086198", "testCaseId": "a8104418aab4ba63e6decb0cab16bf8e", "fullName": "test_div_by_3_6#test_divisible_by_6", "labels": [{"name": "suite", "value": "test_div_by_3_6"}, {"name": "host", "value": "SIVA-PC"}, {"name": "thread", "value": "8312-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_div_by_3_6"}]}